src/jlrichdem/depressions/jldepressions.cpp

```diff
@@ -0,0 +1,30 @@
+#include <jlcxx/jlcxx.hpp>
+#include <jlcxx/stl.hpp>
+#include <typeinfo>
+
+#include <richdem/common/Array2D.hpp>
+#include <richdem/common/constants.hpp>
+#include <richdem/depressions/depression_hierarchy.hpp>
+#include <richdem/depressions/fill_spill_merge.hpp>
+namespace rd = richdem;
+
+JLCXX_MODULE define_depressions_module(jlcxx::Module &mod)
+{
+    jlcxx::TypeWrapper<rd::dephier::Depression<double>> depression_double =
+        mod.add_type<rd::dephier::Depression<double>>("DepressionDouble");
+
+    jlcxx::TypeWrapper<rd::dephier::DepressionHierarchy<double>> depression_hierarchy_double =
+        mod.add_type<rd::dephier::DepressionHierarchy<double>>("DepressionHierarchyDouble");
+
+    mod.method("GetDepressionHierarchyDoubleD8", [](const rd::Array2D<double> &topo,
```

Comment:
Why isn't the method just called `GetDepressionHierarchy`?

Reply:
Just to differentiate it since it is just wrapping the call for Double and D8 parameters. I can definitely change it back to `GetDepressionHierarchy` since it is only in the sense of `double` and `Topology::D8` that it is ever called from the `main.cpp` of fsm repository.